actions/lib/actions.py

```diff
@@ -12,4 +12,9 @@ def _get_client(self):
         username = self.config['username']
         password = self.config['password']
 
-        return sn.Client(instance=instance_name, user=username, password=password)
+        s = sn.Client(instance=instance_name, user=username, password=password)
+
+        if 'custom_params' in self.config:
```

Comment:
Let's also assert `custom_params` is type `dict` before passing it to Service Now. Will make this piece a tad more robust.